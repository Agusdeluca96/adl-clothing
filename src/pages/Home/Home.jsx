import React from 'react';

import { HomeContainer } from './Home.styles'

import Directory from '../../components/Directory/Directory';

const Home = () => (
  <HomeContainer>
    <Directory />
  </HomeContainer>
)

export default Home;