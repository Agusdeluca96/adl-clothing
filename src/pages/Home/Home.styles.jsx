import styled from 'styled-components';

export const HomeContainer = styled.div`
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  height: 100%;

  @media screen and (min-width: 800px) {
    height: -moz-available;          /* WebKit-based browsers will ignore this. */
    height: -webkit-fill-available;  /* Mozilla-based browsers will ignore this. */
    height: fill-available;
  }
`;