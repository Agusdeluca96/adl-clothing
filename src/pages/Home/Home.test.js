import React from 'react';
import { shallow } from 'enzyme';
import Home from './Home';

it('should render Home page component', () => {
  expect(shallow(<Home />)).toMatchSnapshot();
});
