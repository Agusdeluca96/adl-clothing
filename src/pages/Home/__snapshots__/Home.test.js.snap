// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Home page component 1`] = `ShallowWrapper {}`;
